app/views/shared/authentication/_providers_signup_modal.html.erb

```diff
@@ -0,0 +1,13 @@
+<% Authentication::Providers.enabled.each do |provider_name| %>
+  <% provider = Authentication::Providers.get!(provider_name) %>
+
+  <a
+    href="<%= Authentication::Providers.sign_in_path(provider_name, state: "signup-modal") %>"
+    class="sign-up-link cta"
+    data-no-instant>
+    <img
+      src="<%= asset_path("#{provider_name}-logo.svg") %>"
+      class="icon-img"
+      alt="<%= provider.official_name %> logo" /> Auth With <%= provider.official_name %>
```

Comment:
Should this be `Sign In With` rather than `Auth With` for consistency? It looks like we're using both throughout

Reply:
I didn't change the wording from the current version. I'm open to change them obviously but it's long standing copy on the website...

Reply:
Ah, okay. I don't think we should block for that, but at some point if we're in the code again maybe we can make them consistent

Reply:
noted, thanks!